pkg/codegen/nodejs/gen.go

```diff
@@ -2231,6 +2231,7 @@ func (mod *modContext) genResourceModule(w io.Writer) {
 
 	// Check for provider-only modules.
 	var provider *schema.Resource
+	pkgName := mod.pkg.Name()
```

Comment:
Any reason for this change? Or just a clean-up we're bundling in?

Reply:
No, this was a leftover from an earlier attempt.  I've undone it now, since it doesn't really improve anything.